tutorials/Python/00_Introduction/Introduction.ipynb

```diff
@@ -69,35 +51,35 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The 'print' function takes an input value (some text, in quotation marks) and prints the text as output (without the quotation marks). One should note that Python is case-sensitive - ie. we can't use Print or PRINT.\n",
+    "The `print` function takes an input value (some text, in quotation marks) and prints the text as output (without the quotation marks). One should note that Python is case-sensitive - ie. we can't use `Print` or `PRINT`.\n",
     "\n",
-    "Note: As of python 3, the inputs to the 'print' function must be placed within parantheses. If you use code developed for python 2.7 or lower, you may notice that this requirement is not followed and therefore will cause issues if run using python 3+. There are other key differences between python 3 and previous versions, some of which will be covered in this tutorial. For a more detailed look into them, the following resource may be of use: https://docs.python.org/3.6/howto/pyporting.html"
+    "Note: As of python 3, the inputs to the 'print' function must be placed within parantheses. If you use code developed for python 2.7 or lower, you may notice that this requirement is not followed and therefore will cause issues if run using python 3+. There are other key differences between python 3 and previous versions, some of which will be covered in this tutorial. For a more detailed look into them, the following resource may be of use: [Porting Python 2 Code to Python 3](https://docs.python.org/3.6/howto/pyporting.html)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "###### Exercise 0.1:"
+    "### Exercise 0.1:"
```

Comment:
If we are doing a single # for titles, which heading level is using the double ##?

Reply:
`##` is the section (but it might be incorrect somewhere). For example:
```
# 1. Object types in Python

## 1.1 Strings

### Exercise 1
Exercise 1 on strings.
```

Reply:
Ok cool thanks!